src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { User } from '../../types/User';
+
+interface Props {
+  onAddTodo: (title: string, userId: number) => void;
+  users: User[];
+}
+
+export const TodoForm: React.FC<Props> = ({ onAddTodo, users }) => {
+  const [selectUserId, setSelectUserId] = useState(0);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [hasErrorSelectUserId, setHasErrorSelectUserId] = useState(false);
+  const [hasErrorTodoTitle, setHasErrorTodoTitle] = useState(false);
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    if (selectUserId === 0) {
+      setHasErrorSelectUserId(true);
+    }
+
+    if (todoTitle === '') {
+      setHasErrorTodoTitle(true);
+    }
+
+    if (selectUserId === 0 || todoTitle === '') {
```

Comment:
- You do a comparison  twice, create variables
- You don't need to compare with an empty string, just check if a value exists